src/main/java/pb360/data/repository/MessageObjectRepository.java

```diff
@@ -1,9 +1,18 @@
 package pb360.data.repository;
 
+import java.util.List;
+
 import org.springframework.data.mongodb.repository.MongoRepository;
 
+import pb360.data.entity.RestApi;
 import pb360.model.MessageObject;
 
-public interface MessageObjectRepository extends MongoRepository<MessageObject, String> {
+public interface MessageObjectRepository extends MongoRepository<RestApi, String> {
+	
+//	public RestApi findBymessageNumber(String messageNumber);
+//	public RestApi findFirstByOrderByrestIdDesc();
+	public int countByrestId();
```

Comment:
change return type to Integer, sometime you will get null when DB is empty, int cannot contain a null value